src/cookieManager.ts

```diff
@@ -63,13 +85,7 @@ export class CookieManager {
       ';expires=Tue, 1 Jan 1980 23:59:59 GMT';
   }
   private getCookie() {
-    const docCookie = document.cookie;
-    if (docCookie) {
-      const payload = docCookie.split('; ').find((row) => row.startsWith(this._cookieName + '='));
-      if (payload) {
-        return decodeURIComponent(payload.split('=')[1]);
-      }
-    }
+    return getCookie(this._cookieName);
```

Comment:
Not sure if this is conventional or not to have these two functions be the same name - happy to change the name if needed

Reply:
I don't love it.  It obviously works but it adds a little extra confusion.